### DISCUSSION TITLE
LoTW Certificate Upload Problems

### DISCUSSION THREAD

**Discussion by ychulin** [magicbug/Cloudlog#1192] [General]
I tried to upload a p12 file, but the same error message every time, help me find out what is the reason?
![20210921165916](https://user-images.githubusercontent.com/42275318/134142747-7ca66f0b-93a9-49ed-9abd-55dc2752d86b.png)

**Comment by AndreasK79** [comment]
Is this a new install? If so, did you run Admin -> Update Country Files?

**Reply by ychulin** [reply]
yes ,is new install.
Admin -> Update Country Files 
Has been completed，and problem still occurs.

Here you can update the DXCC lookup data that is used for displaying callsign information.

This data is provided by Clublog.

Check for DXCC Data Updates
DONE
Dxcc Entities: 402
Dxcc Exceptions: 23414
Dxcc Prefixes: 4387

**Comment by magicbug** [comment]
That's definitely an issue with the Update Countrys can you create Station Locations ok?

**Reply by ychulin** [reply]
Yes, sure
In the Station DXCC option here
I can select the correct data and the correct update station profile

**Comment by ychulin** [comment]
I deleted the entire URL including the database
Then follow the steps below
and problem still occurs.

Setup as always with any project is a bit of a pinch point and there seems to be a recurring issue of people not reading the install wiki page fully, therefore time to build a checklist for new setups.

This will show up when you hit the dashboard if you haven't carried out the tasks

Update Country Files to populate the database
Create a new user and delete the demo account
Create a new station profile and set it to active

**Comment by agrif** [comment]
This is a very late update, but maybe it will help.

I ran into an issue that looks exactly like this, and there are three ways to fix it:

 * The changes in pull request #1772.
 * Install the `file` command line utility.
 * Install the `php7-fileinfo` module.
